package.json

```diff
@@ -28,17 +27,11 @@
     "@rdfjs/formats-common": "^2.2.0",
     "express": "^4.18.2",
     "hijackresponse": "^5.0.0",
-    "lit": "^2.3.1",
-    "rdf-ext": "^2.0.1"
+    "rdf-ext": "^2.0.1",
+    "rdf-entity-webcomponent": "^0.2.0"
```

Comment:
Can we make sure that in the future we publish new packages in the `@zazuko/` namespace?

Reply:
Can one use the namespace for specific things also?

for example `@zazuko/trifid`?

Reply:
For Trifid, either `@zazuko` or `@trifid`.
This is something that we might need to discuss at some point.